hack/generated/apis/microsoft.batch/v1alpha1api20210101/batch_account_types_gen.go

```diff
@@ -76,6 +76,16 @@ func (batchAccount *BatchAccount) AzureName() string {
 	return batchAccount.Spec.AzureName
 }
 
+// GetSpec returns the specification of this resource
+func (batchAccount *BatchAccount) GetSpec() genruntime.ConvertibleSpec {
+	return &batchAccount.Spec
+}
+
+// GetStatus returns the current status of this resource
```

Comment:
minor: To match the comment above:
```suggestion
// GetStatus returns the status of this resource
```

Or change the above one to say "current"

Reply:
Updated the comments to be consistent.